src/frontend/js/components/Search/index.tsx

```diff
@@ -1,36 +1,24 @@
-import { Fragment, useState, useEffect } from 'react';
```

Comment:
You should pass `onFiltersPaneCloseButtonClick` to the `onClick` of overlay (L179) because currently when user touch the overlay the filter pane is not close smoothly.

Reply:
Good catch, thanks!